packages/langium/src/index.ts

```diff
@@ -7,8 +7,8 @@
 export * from './default-module';
 export * from './dependency-injection';
 export * from './generator/generator-node';
-export * from './generator/node-processor';
-export * from './generator/template-string';
+export { expandToNode, forEach } from './generator/template-node';
+export { expandToString, expandToStringWithNL } from './generator/template-string';
```

Comment:
Why not export the whole file contents? Do they contain stuff that we use only internally?

Reply:
Changed it export everything from `template-node.ts` (restriction was not necessary there).

`template-string.ts` contains definitions that are used internally in other modules, but shall not be published globally, at least not now.